library/src/test/java/com/digi/xbee/api/models/HardwareVersionEnumTest.java

```diff
@@ -77,7 +77,11 @@ public void testHardwareVersionEnumDescriptions() {
 	 */
 	@Test
 	public void testHardwareVersionEnumStaticAccess() {
-		for (HardwareVersionEnum hardwareVersionEnum:hardwareVersionValues)
+		for (HardwareVersionEnum hardwareVersionEnum:hardwareVersionValues) {
+			// Do not check deprecated entry.
+			if (hardwareVersionEnum == HardwareVersionEnum.CELLULAR)
```

Comment:
Maybe you can check the `CELLULAR` enumeration but with `CELLULAR_CAT1_LTE_VERIZON`. Something like:
```
if (hardwareVersionEnum == HardwareVersionEnum.CELLULAR) {
	HardwareVersionEnum hv = HardwareVersionEnum.get(hardwareVersionEnum.getValue())
	assertEquals(hv.getDescription(),  HardwareVersionEnum.CELLULAR_CAT1_LTE_VERIZON);
	continue;
}
```